index.js

```diff
@@ -150,7 +150,7 @@ function parse(input, options) {
 	}
 
 	for (const param of input.split('&')) {
-		let [key, value] = param.replace(/\+/g, ' ').split('=');
+		let [key, value] = param.replace(/\+/g, ' ').replace('=', '<**>').split('<**>');
```

Comment:
The previous regex was better. This is risky as it will fail if the query string actually contains `<**>`, even if unlikely.